CHANGELOG.md

```diff
@@ -12,6 +12,27 @@
 * Add new feature gate `ServerSideApplyPhase1` (disabled by default) that adds support for Server Side Apply for `ConfigMap`, `Ingress`, `PVC`, `Service`, and `ServiceAccount` according to [Strimzi Proposal #105](https://github.com/strimzi/proposals/blob/main/105-server-side-apply-implementation-fg-timelines.md).
 * Added distinction between changes of "cluster-wide" broker properties applied dynamically at cluster level, and "per-broker" broker properties applied dynamically at broker level.
 * Extend the EntityOperator, Cruise Control and KafkaExporter deployment to support PDB via the template section in the CR spec.
+* Fix RBAC naming for **KafkaMirrorMaker2** to avoid Role/RoleBinding collisions when a **KafkaConnect** with the same name exists in the same namespace. KafkaMirrorMaker2 now uses dedicated Role/RoleBinding names.
```

Comment:
I think this should be moved tot he topp of the major changes section: https://github.com/strimzi/strimzi-kafka-operator/blob/main/CHANGELOG.md#major-changes-deprecations-and-removals